source/Calamari.Tests/Fixtures/Integration/Scripting/CSharpScriptEngineFixture.cs

```diff
@@ -21,7 +22,8 @@ public void DotnetScript_CSharpDecryptsVariables()
                 var variables = GetVariables();
                 variables.Add(ScriptVariables.UseDotnetScript, bool.TrueString);
                 File.WriteAllText(scriptFile.FilePath, "System.Console.WriteLine(OctopusParameters[\"mysecrect\"]);");
-                var result = ExecuteScript(new DotnetScriptExecutor(), scriptFile.FilePath, variables);
+                var commandLineRunner = new TestCommandLineRunner(new InMemoryLog(), new CalamariVariables());
+                var result = ExecuteScript(new DotnetScriptExecutor(commandLineRunner), scriptFile.FilePath, variables);
```

Comment:
I tried testing this, including a bundled dotnet-script package, extracting that to a path, and adding that path to the environment variables. In the test context this was not retrieving the executable path on the `where/which` commands. I sunk a lot of time into this and couldn't work out why this wasn't working.